### PR TITLE
Fixes #21177 - missing warning icon on ssh keys

### PR DESCRIPTION
**Pull request by boaz0**
### Before
before
### After
screenshot_2018-09-06 screenshot

### CODE REVIEW

**Comment by boaz0** [comment]
[test foreman]

**Comment by ohadlevy** [comment]
I wonder why do you have my account in your screenshare ;)

File: app/views/ssh_keys/_form.html.erb [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <div class="alert alert-warning alert-block base">
       <% [:fingerprint, :length].each do |field|%>
         <% @ssh_key.errors.full_messages_for(field).each do |e| %>
-          <li><%= e %></li>
+          <li><span class="glyphicon glyphicon-exclamation-sign"></span> <%= e %></li>
```
Comment (ohadlevy) [inline app/views/ssh_keys/_form.html.erb]: i believe we have an helper for icons?

**Comment by boaz0** [comment]
@ohadlevy don't worry I didn't use any exploits to compromise your machine. I just used the screenshot you uploaded to the issue tracker. :smile:

**Comment by boaz1337** [comment]
[test foreman]

**Comment by boaz1337** [comment]
[test continuous-integration/travis-ci/pr]

**Comment by ares** [comment]
Do we display such icons in other forms? I can't recall I would see that, but perhaps things have changed.

**Comment by mmoll** [comment]
[test foreman]
[test upgrade]
[test katello]

**Comment by Rohoover** [comment]
@ares 
Yea I don't think that is the correct icon:
https://www.patternfly.org/pattern-library/communication/inline-notifications/

Warning: pficon-warning-triangle-o
Error: pficon-error-circle-o

**Comment by boaz0** [comment]
Thanks @ares @Rohoover 
I have updated the code to use `pficon-warning-triangle-o`

**Comment by ares** [comment]
Sorry I was probably not clear, what I meant is that we don't display any icon in error bugs on other pages I think.

see these examples
error2
error1

**Comment by Rohoover** [comment]
@ares 

Do you mean per line item within the inline box?

**Comment by ares** [comment]
> Do you mean per line item within the inline box?

yeah, those little triangles left of messages. When we have alert box, there's just one icon for the whole box, representing whether this is error or warning box. On my screens you see multiline message but there's single red cross for the whole box. Not an icon for each message in that box.

**Comment by Rohoover** [comment]
@ares 
Yes I believe you are correct.

**Comment by boaz0** [comment]
Thanks. I updated the code again. Let me know what you think.

**Review by ares — APPROVED** [review]
Ok, this is exactly what I meant, thank you. When I tested I'm a bit confused why we display the alert here. In other forms we just mark fields with red error messages which we do here too. See any other form, e.g. 
err

Anyway I know it's not the code you introduced so I'm happy to merge as is and set separate PR to remove the alert box completely. Also the reason here might be that parsing SSH key may fail due to various reason which is why we display more info in warning box.

@Rohoover any insights or good to merge?

**Comment by Rohoover** [comment]
I guess I'm a little confused so these are my thoughts:

1) One icon for the entire inline message box.

2)Form level validation errors should be restricted to the form itself and no an inline message.

**Comment by ohadlevy** [comment]
@boaz1337 I assume the screenshot is updated, @Rohoover can you have another look?

**Comment by ares** [comment]
Sorry this was pending on my side, I think Rox did not have any other suggestions. I guess we're good to merge. I think the box does not have to be displayed at all, we don't display it elsewhere, but the current solution is acceptable for this PR.

Thanks @boaz1337, merging!

**Pull request merged**